src/log.tf

```diff
@@ -0,0 +1,48 @@
+resource "azurerm_storage_account" "diag-log" {
```

Comment:
It might be good for us to break this out into a module. Is this for compliance, would we follow this pattern again of having an azure storage for logs?